substra/cli/printers.py

```diff
@@ -128,6 +128,19 @@ def get_value(self, item, expand=False):
         return f'{done_count}/{tuple_count}'
 
 
+class MappingField(Field):
+    def get_value(self, item, expand=False):
+        mapping = super().get_value(item) or {}
+        if expand:
+            value = [
+                f'{key}:{mapping[key]}'
+                for key in mapping.keys()
```

Comment:
```suggestion
                for key, v in mapping.items()
```

Minor: no need to access the value using `mapping[key]`.

Reply:
Very good point!